pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -349,7 +349,10 @@ def make_package(args):
     if args.icon_fg and args.icon_bg:
         shutil.copy(args.icon_fg, join(res_dir, 'mipmap/icon_foreground.png'))
         shutil.copy(args.icon_bg, join(res_dir, 'mipmap/icon_background.png'))
-        with open(join(res_dir, 'mipmap-anydpi-v26/icon.xml'), "w") as fd:
+        mipmap_anydpi = join(res_dir, 'mipmap-anydpi-v26')
```

Comment:
I agree with @interlark, `anydpi_dir` is a better name for this variable.
```suggestion
        anydpi_dir = join(res_dir, 'mipmap-anydpi-v26')
```

Reply:
In the new commit I've used the `anydpi_dir` variable name.